inference_cli/lib/roboflow_cloud/batch_processing/core.py

```
@@ -0,0 +1,356 @@
+from typing import List, Optional
+
+import typer
+from typing_extensions import Annotated
+
+from inference_cli.lib.env import ROBOFLOW_API_KEY
+from inference_cli.lib.roboflow_cloud.batch_processing.api_operations import (
+    display_batch_job_details,
+    display_batch_jobs,
+    trigger_job_with_workflows_images_processing,
+    trigger_job_with_workflows_videos_processing,
+)
+from inference_cli.lib.roboflow_cloud.batch_processing.entities import (
+    AggregationFormat,
+    MachineSize,
+    MachineType,
+)
+
+batch_processing_app = typer.Typer(
+    help="Commands for interacting with Roboflow Batch Processing"
+)
+
+
+@batch_processing_app.command(help="List batch jobs in your workspace.")
+def list_jobs(
+    api_key: Annotated[
+        Optional[str],
+        typer.Option(
+            "--api-key",
+            "-a",
+            help="Roboflow API key for your workspace. If not given - env variable `ROBOFLOW_API_KEY` will be used",
+        ),
+    ] = None,
+    max_pages: Annotated[
+        Optional[int],
+        typer.Option(
+            "--max-pages",
+            "-m",
+            help="Number of pagination pages with batch jobs to display",
+        ),
+    ] = 1,
+    debug_mode: Annotated[
+        bool,
+        typer.Option(
+            "--debug-mode/--no-debug-mode",
+            help="Flag enabling errors stack traces to be displayed (helpful for debugging)",
+        ),
+    ] = False,
+) -> None:
+    if api_key is None:
+        api_key = ROBOFLOW_API_KEY
```

Maybe it would be good to throw early if `ROBOFLOW_API_KEY` is not set